kalite/contentload/management/commands/channels/base.py

```diff
@@ -129,8 +129,16 @@ def recurse_nodes(node, path=""):
             try:
                 if child_datum["kind"] == "Exercise":
                     child_denormed_data = exercise_lookup[str(child_datum["id"])]
+                    # Add path information here
+                    slug = exercise_lookup[str(child_datum["id"])][channel_data["slug_key"][child_datum["kind"]]] if exercise_lookup[str(child_datum["id"])][channel_data["slug_key"][child_datum["kind"]]] != "root" else "khan"
+                    slug = slugify(unicode(slug))
+                    exercise_lookup[str(child_datum["id"])]["path"] = node["path"] + slug + "/"
                 elif child_datum["kind"] == "Video":
                     child_denormed_data = content_lookup[str(child_datum["id"])]
+                    # Add path information here
+                    slug = content_lookup[str(child_datum["id"])][channel_data["slug_key"][child_datum["kind"]]] if content_lookup[str(child_datum["id"])][channel_data["slug_key"][child_datum["kind"]]] != "root" else "khan"
```

Comment:
Same here, looks like their only difference is this uses `content_lookup` instead of `exercise_lookup`. Any way we can combine these two?